src/riak_repl2_fssink.erl

```diff
@@ -47,7 +47,7 @@
         cluster,
         fullsync_worker,
         work_dir = undefined,
-        strategy :: keylist | aae,
+        strategy :: keylist | aae | undefined,
```

Comment:
The documentation says:
```
%% If the protocol version >= 2.0, and the cluster has aae enabled and the replication
%% configuration stanza enables the aae strategy (the default is keylist),
```
In other words, it assumes a default. I would expect the init function to then implement that default by initialising the state with `keylist` alternatively to follow the logic.

At this moment there is a possible race condition when the server is started and a call `legacy_status` comes in, this may crash the server (line 107).  Dializer seems not to spot this, though.